lib/message_sender.js

```diff
@@ -37,6 +37,30 @@ MessageSender.prototype = {
             });
     },
 
+    /**:MessageSender.create_outbound_message(outbound_msg_info)
+    Creates the outbound messages to seed-message-sender
+
+    :param object outbound_msg_info:
+      Gets passed through directly to service. Consult service docs/protocol.
+      Example object content:
+            {
+                "to_addr": "+27123",
+                "identity": identity uuid,
+                "content": "my outbound message",
+                "metadata": {}
+            }
+    */
+    create_outbound_message: function(outbound_msg_info) {
```

Comment:
please make it params again with this one. `metadata` can be none/undefined on but if so should be set to `{}` before posting.